### PR TITLE
documentation: S3C-4319-milestone-2-testing-steps

### PR DESCRIPTION
**Pull request by anurag4DSB**
How to test Milestone-2

### CODE REVIEW

File: docs/milestone-2-steps.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+### Prepare OSIS machine
+
+- Use a stateless node of S3 Connector to run the OSIS docker container. If stateless and stateful nodes are on the same storage nodes in the lap, you can deploy it there.
+
+- Login to docker registry:
+
+```sh
+# You will need access to `registry.scality.com/vmware-ose-scality-dev`. For credentials and access to the image repository please contact it@scality.com.
+docker login -u <username> registry.scality.com
```
Comment (saitharunthodupunuru) [inline docs/milestone-2-steps.md:72]: I think our image is open source as our code is. May be we don't need docker login at all.

File: docs/milestone-2-steps.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+      spring.redis.ssl=false
+      spring.redis.timeout=60000
+      spring.redis.password=
+      spring.redis.sentinel.master=scality-s3
+      spring.redis.sentinel.nodes=10.100.2.202:16379, 10.100.6.138:16380, 10.100.2.80:16379
+      spring.redis.lettuce.shutdown-timeout=200ms
+      osis.scality.redis.credentials.hashKey=s3credentials
+      ```
+
+- Run OSIS docker container by following instructions [here](https://github.com/scality/vmware-ose-scality/blob/development/1.0/src/main/resources/crypto.yml)
```
Comment (saitharunthodupunuru) [inline docs/milestone-2-steps.md:105]: Wrong link

File: docs/milestone-2-steps.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
+# Deployment Instructions
+
+## Limitations
+
+- Bucket versioning not supported: [S3C-4456](https://scality.atlassian.net/browse/S3C-4456)
+- Buckets and objects not created via VMware OSE do not appear in vCloud Director: [S3C-4199](https://scality.atlassian.net/browse/S3C-4199)
+
+## Prerequisite
```
Comment (saitharunthodupunuru) [inline docs/milestone-2-steps.md:8]:
```suggestion
## Prerequisites
```

**Review by saitharunthodupunuru — CHANGES_REQUESTED** [review]
Minor changes. I am good with everything else.
`Use Scality Application with VMWare Object Storage extension` part is awesome and the screenshots will always be of great help.

May be if allowed can we add the google drive link of the demo video erwan prepared or the original video I created for better understanding?

File: docs/milestone-2-steps.md (hunk 3 above)
Comment (anurag4DSB) [reply docs/milestone-2-steps.md]: Changed in squash commit: [`bc337ec` (#52)](https://github.com/scality/vmware-ose-scality/pull/52/commits/bc337ec87d8758d0be6cdf87ef99dd6f005236dc)

File: docs/milestone-2-steps.md (hunk 1 above)
Comment (anurag4DSB) [reply docs/milestone-2-steps.md]: Yes, you are right, removed here: [`bc337ec` (#52)](https://github.com/scality/vmware-ose-scality/pull/52/commits/bc337ec87d8758d0be6cdf87ef99dd6f005236dc)

File: docs/milestone-2-steps.md (hunk 2 above)
Comment (anurag4DSB) [reply docs/milestone-2-steps.md]: Corrected: [`bc337ec` (#52)](https://github.com/scality/vmware-ose-scality/pull/52/commits/bc337ec87d8758d0be6cdf87ef99dd6f005236dc)

**Comment by anurag4DSB** [comment]
> Minor changes. I am good with everything else.
> `Use Scality Application with VMWare Object Storage extension` part is awesome and the screenshots will always be of great help.
> 
> Maybe if allowed can we add the google drive link of the demo video erwan prepared or the original video I created for better understanding?

Made the changes.
The video is being distributed by Product team for demo purposes. Does not belong here in the documentation. If we need the video to explain this doc, then this documentation needs to be improved. Open to suggestions, if you see any other improvement. :)

**Review by saitharunthodupunuru — APPROVED** [review]

**Review by ilkescality — APPROVED** [review]

**Pull request merged**